.gitignore

```diff
@@ -1,3 +1,4 @@
 /out
 /cache
 *.log
+.env
```

Comment:
why this was included? It's not part of the current local development setup afaik.

Reply:
Probably should be removed